gyronimo/dynamics/curvilinear_boris.hh

```diff
@@ -26,62 +26,76 @@
 
 namespace gyronimo {
 
-//! Gyronimo implementation for curvilinear boris stepper class.
+//! Boris-like stepper, cartesian velocity and curvilinear position.
 /*!
-        This class implements the curvilinear boris pusher
-        [Journal of Plasma Physics, Volume 61, Issue 3, April 1999, pp. 367 -
-   389]. This class mixes the typical Cartesian boris push with a Runge-Kutta 2
-   advance in curvilinear coordinates. To achieve this, field representation is
-   only constrained to metrics of the type `metric_connected` to be able to
-   convert coordinates and vectors using the connected `morphism`. To use the
-   classical boris pusher in curvilinear coordinates, look at `classical_boris`
-   class.
+    Advances a charged-particle's cartesian velocity by a single
+    time step using the Boris algorithm [C. K. Birdsall and A. B. Langdon,
+    Plasma Physics via Computer Simulation, CRC Press, 1991], which is a
```

Comment:
Should we mention that this algorithm was taken from Delzanno's paper of 2013 (even though it is Boris-like)?

Reply:
The 2013 paper is mentioned a little later in the documentation, when describing the curvilinear position step.